### ISSUE TITLE
Stackshare tool

### ISSUE THREAD

**Issue by GandelXIV** [Keats/gutenberg#2182]
Any plans on adding this tool to https://stackshare.io/? I would like to include it to some of our stacks.

**Comment by Keats** [comment]
Never heard about that. I'm not going to sign up to add it though, I'm guessing anyone can add it?

**Comment by GandelXIV** [comment]
Yes, but it requires some info about the tool, so it would be better for some of the contributors to do it.

**Comment by GandelXIV** [comment]
I can add it if you do not wish to create an account, but i am going to need to know which info to enter:
![image](https://user-images.githubusercontent.com/61948151/234833500-1582d7e7-081f-42b9-a97d-4386bd7e42d4.png)
Note that `zola` is already taken as name of tool

**Comment by GandelXIV** [comment]
Also an icon

**Comment by Keats** [comment]
What's the other Zola tool? For the info you can just take things from https://www.getzola.org/ and the favicon as the icon

**Comment by GandelXIV** [comment]
I think its a company

**Comment by Keats** [comment]
Ah just saw https://www.zola.fr/ maybe it's that?
